lms/urls.py

```diff
@@ -33,6 +33,11 @@
     url(r'', include('student.urls')),
     # TODO: Move lms specific student views out of common code
     url(r'^dashboard$', 'student.views.dashboard', name="dashboard"),
+
+    url(r'^course/{}/refund_status$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
Again, you are getting data about a specific enrollment, not a course run. Enrollment are refundable. Course runs are not. At a minimum this needs to be out of the root urls.py file. adding more individual routs to this file is adding more debt. 

Move this route to either the enrollments API or nested under the dashboard routes.